ynh-dev

```diff
@@ -365,11 +383,14 @@ with open(setting_file) as f:
 EOF
                 fi
 
-                local DISTRO="$(lsb_release -s -c)"
-
                 cd "$DEV_PATH"
+                systemctl start yunohost-api
```

Comment:
That's weird. When working with the container, yunohost-api seems not started and it creates problems when developing the front.